SensorDict.py

```diff
@@ -0,0 +1,70 @@
+import csv
+
+# Import sensor data from CSF file. This file contains information
+# about each individual sensor on the car as well as its can ID
+# and data offset used to parse the specific sensor's data
+
+sensor_file = csv.DictReader(open('SensorDict.csv'))
+sensor_data = csv.DictWriter(open('SensorData.csv', "a"), fieldnames=[
+                             "timestamp", "id", "data"], extrasaction="ignore")
+# sensor_data will APPEND to SensorData.csv with fields timestamp,id,data
+
+# Takes in a dictionary file with the format for all of the sensor data.
+# Also takes in a dictionary of data with format:
+# {"timestamp" : "XXX", "id" : "XXX", "data" : "XXX"}
+# Returns a dictionary with the data split up with corresponding names.
+
+
+def data_to_dict(dict_file, data):
+    id = "0x" + data["id"]
+    message = data["data"]
+    timestamp = data["timestamp"]
+    output_dict = {}
+    for row in dict_file:
+        if (row["id"] != id):
+            continue
+        lsb = int(row["LSB"])
+        # Least significant byte
```

Comment:
I think this can be written better, see the [pep8](https://www.python.org/dev/peps/pep-0008/#comments) guide for comments. I think comments should either be inline or above the statement. We could also rename the variables to make it more clear